frame/support/procedural/src/pallet/parse/mod.rs

```diff
@@ -425,7 +427,36 @@ enum PalletAttr {
 	Config(proc_macro2::Span),
 	Pallet(proc_macro2::Span),
 	Hooks(proc_macro2::Span),
-	RuntimeCall(proc_macro2::Span),
+	/// A `#[pallet::call]` with optional attribute.
```

Comment:
Is there a better spot to document this?  
I feel like the only way would be to create a new macro…

Reply:
I tried it yesterday. And for whatever reason the rust lang server did not like to highlight the `pallet::call` even with a stub…  
Dont know if i did it wrong, but it is annoying.